plugins/formal-verification/src/org/jetbrains/kotlin/formver/scala/silicon/ast/Stmt.kt

```diff
@@ -36,6 +36,22 @@ sealed class Stmt : IntoViper<viper.silver.ast.Stmt> {
             viper.silver.ast.FieldAssign(lhs.toViper() as FieldAccess, rhs.toViper(), position.toViper(), info.toViper(), trafos.toViper())
     }
 
+    companion object {
+        fun assign(
+            lhs: Exp,
+            rhs: Exp,
+            position: Position = Position.NoPosition,
+            info: Info = Info.NoInfo,
+            trafos: Trafos = Trafos.NoTrafos,
+        ): Stmt = when (lhs) {
+            is Exp.LocalVar ->
+                LocalVarAssign(lhs, rhs, position, info, trafos)
+            is Exp.FieldAccess ->
+                FieldAssign(lhs, rhs, position, info, trafos)
```

Comment:
I was saying, since this case is not tested yet (I think because now we cannot do that), maybe we can add a "TODO" line as a reminder, but probably is useless :)

Reply:
Ah, since we only do end-to-end testing I think it's probably fine; it'll happen automatically when we implement fields :)